packages/availity-react-kit/project/app/Request/components/Patient.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { observer } from 'mobx-react';
+import { AvField } from '../../components';
+import { Label, UncontrolledTooltip } from 'reactstrap';
+import PropTypes from 'prop-types';
+import { RequestStore } from '../stores/requestStore';
+
+@observer
+export default class Patient extends Component {
+  constructor(props) {
+    super(props);
+
+    this.updateMemberId = ::this.updateMemberId;
+  }
+
+  updateMemberId(event) {
+    const { requestStore } = this.props;
+    const { value } = event.target;
+
+    requestStore.updateMemberId(value);
+  }
+
+  render() {
+    const { dob, memberId } = this.props.requestStore;
+
+    return (
+      <fieldset>
+
+        <legend>Member</legend>
+
+        <Label id="memberid-help" for="memberID">Member ID
+          <span className="inline-help">What's this?</span>
+        </Label>
+        <UncontrolledTooltip
+          target="memberid-help"
+          placement="top"
+        >Also known as subscriber ID or member number
+        </UncontrolledTooltip>
+
+        <AvField
+          type="text"
+          id="memberID"
+          name="memberID"
+          value={memberId}
+          validate={{
+            number: {value: true, errorMessage: 'Must be a number'},
+            minLength: {value: 5, errorMessage: '5 Character Minimum'}
+          }}
+        />
+
+        <AvField
+          type="text"
```

Comment:
since this is a date, should this be `type="date"`?

Reply:
Good catch i'll fix